content/en/docs/concepts/storage/volumes.md

```diff
@@ -865,6 +870,11 @@ A projected volume maps several existing volume sources into the same
 directory. For more details, see [projected volumes](/docs/concepts/storage/projected-volumes/).
 
 ### rbd
+{{< feature-state for_k8s_version="v1.28" state="deprecated" >}}
+
+{{< note >}}
+We recommend to use Ceph RBD CSI out-of-tree driver instead.
```

Comment:
We [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we). Please reword, eg “The Kubernetes project recommends&hellip;”